ee/clickhouse/sql/events.py

```diff
@@ -21,14 +21,15 @@
     created_at DateTime64(6, 'UTC')
     {materialized_columns}
     {extra_fields}
-) ENGINE = {engine} 
+) ENGINE = {engine}
 """
 
 EVENTS_TABLE_MATERIALIZED_COLUMNS = """
-    , properties_issampledevent VARCHAR materialized trim(BOTH '\"' FROM JSONExtractRaw(properties, 'isSampledEvent'))
```

Comment:
Do we run into backwards incompatibility issues & schema sync issues here with Cloud?